src/ensemble/basic_ensemble_solve.jl

```diff
@@ -23,6 +23,20 @@ $(TYPEDEF)
 """
 struct EnsembleSerial <: BasicEnsembleAlgorithm end
 
+function merge_stats(us)
+    st = Iterators.filter(!isnothing, (x.stats for x in us))
+    try
+        return reduce(merge, st)
+    catch e
+        if isa(e, MethodError)
+            # there were no stats or they didn't have a merge method
+            return nothing
+        else
+            rethrow(e)
+        end
+    end
```

Comment:
Since this is managed here now, we can just assume that the stats have a merge function because all current stats have it, and any that would be added in the future just need to make sure they add it. An error is better than hiding.
```suggestion
    reduce(merge, st)
```

Reply:
Oh I see, that line is still there, nvm